pkg/process/checks/process_nix.go

```diff
@@ -19,6 +19,10 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/system"
 )
 
+var (
+	hostCPUCount = system.HostCPUCount
```

Comment:
mind adding a short comment for the intention here (used for tests)

Reply:
Done!